docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: "2"
+services:
+  dasher:
+    build: .
+    network_mode: host
+    privileged: true
```

Comment:
Is `--privileged` really necessary?
I'm running [clemenstyp/dasher-docker](https://hub.docker.com/r/clemenstyp/dasher-docker/) on my system without it and it is working fine.